apluslms_roman/cache_file.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from os import mkdir, remove
+from os.path import isdir, join
+
+from apluslms_yamlidator.document import Document
+from apluslms_yamlidator.validator import ValidationError
+from apluslms_yamlidator.utils.version import Version
+
+from . import CACHE_DIR
+
+class CacheFile(Document):
+    name = 'roman_cache'
+    schema = name
+    version = Version(1, 0)
+    path = join(CACHE_DIR, '%s.yml' % name)
```

Comment:
You could set `version_key = None` and then this Document, shouldn't store version number in the file. As the file is removew, if the validation failes, the old version file should be automatically removed. This also means, there wouldn't be `version` in the schema.

This is ok for cache files as it's valid to remove them when not needed.

Reply:
it does actually save the version number even if you put `version_key = None`?